### PR TITLE
Update SEC landing page with list of participating orgs and partner orgs

### PR DESCRIPTION
**Pull request by zoebentley**
Updates the page to match this document: 
https://docs.google.com/document/d/1929SgqY91JTC2Ds35tNdgI3aJh5ZkZCvibT3tOaMJwY/edit#

Overview of changes: 
- Restructure page into 2 sections, "Participating Orgs" and "Partner Orgs"
- Add 40 participating orgs (with links to their websites, and locations of each organization) 
- Add 6 partner orgs
- Put wording at bottom that says the application deadline has passed

### CODE REVIEW

File: src/views/sec/l10n.json [hunk 1]
```diff
@@ -1,18 +1,124 @@
 {
-  "sec.title": "Scratch Education Collaborative",
-  "sec.intro": "Join a global network of organizations supporting creative coding",
+  "sec.title": "Scratch Education Collaborative (SEC)",
+  "sec.intro": "Participating Orgs",
```
Comment (apple502j) [inline src/views/sec/l10n.json:3]: Why abbreviate?

File: src/views/sec/l10n.json [hunk 2]
```diff
@@ -1,18 +1,124 @@
...
-  "sec.applyDeadline": "The deadline for applying to the Scratch Education Collaborative has been extended to March 1st, 2021",
+  "sec.applyBanner": "Applications are now closed. The deadline for applying to the SEC for 2021 was March 1st.",
+  "sec.projectsTitle": "Participating Orgs",
+  "sec.yearRange": "2021-2022",
+  "sec.eligibilityTitle": "Elgibility",
+  "sec.applyTitle": "Applications are now closed",
+  "sec.applyDeadline": "The deadline for applying to the SEC for 2021 was March 1st.",
   "sec.applyButton": "Click here to apply",
+  "sec.org1": "{org1Listing} - AK, USA",
+  "sec.org1LinkText": "Alaska Native Science & Engineering Program",
```
Comment (apple502j) [inline src/views/sec/l10n.json:13]: I don't think organization names should be translated.

File: src/views/sec/l10n.json [hunk 3]
```diff
@@ -1,18 +1,124 @@
...
+  "sec.org36LinkText": "Teachathon Foundation",
+  "sec.org37": "{org37Listing} - Girona, Spain",
+  "sec.org37LinkText": "UdiGitalEdu",
+  "sec.org38": "{org38Listing} - Halifax, Nova Scotia",
+  "sec.org38LinkText": "Ulnooweg - Digital Mi'kmaq",
+  "sec.org39": "{org39Listing} - USA",
+  "sec.org39LinkText": "WeTeach_CS, Center for Cyber Education at University of Texas Austin",
+  "sec.org40": "{org40Listing} - Owerri, Nigeria & Nairobi, Kenya",
+  "sec.org40LinkText": "Youth for Technology Foundation",
+  "sec.letterA": "A",
```
Comment (apple502j) [inline src/views/sec/l10n.json:92]: And those letters; they should not be translated.

File: src/views/sec/l10n.json (hunk 2 above)
Comment (zoebentley) [reply src/views/sec/l10n.json:13]: Sounds good -- just changed

File: src/views/sec/l10n.json (hunk 3 above)
Comment (zoebentley) [reply src/views/sec/l10n.json]: Just changed

**Review by seotts — APPROVED** [review]
Looks good!

**Pull request merged**